Library/Homebrew/dev-cmd/update-python-resources.rb

```diff
@@ -37,6 +37,10 @@ def update_python_resources_args
   def update_python_resources
     args = update_python_resources_args.parse
 
+    # Python packages may need access to additional commands while generating pip report.
+    # We use the same PATH as bump commands to make sure resource updates behave similarly.
+    ENV["PATH"] = PATH.new(ORIGINAL_PATHS).to_s
```

Comment:
```suggestion
    with_env(PATH: PATH.new(ORIGINAL_PATHS).to_s) do
```

and then you'll need an `end` below to restore the path like @carlocab mentioned.

Reply:
Looks like we may have helper. May use that if it works:
https://github.com/Homebrew/brew/blob/6c6c418dffe6a227d326514891f27b4b994eb317/Library/Homebrew/extend/kernel.rb#L230-L232